PatchesAndAreasApi.Tests/V1/E2ETests/Stories/ReplacePatchResponsibleEntitiesTests.cs

```diff
@@ -28,8 +28,15 @@ public ReplacePatchResponsibleEntitiesTests(MockWebApplicationFactory<Startup> a
             _snsFixture = appFactory.SnsFixture;
             _patchFixture = new PatchesFixtures(_dbFixture.DynamoDbContext, _snsFixture.SimpleNotificationService);
             _steps = new ReplacePatchResponsibleEntitiesStep(appFactory.Client);
+
+            // AppContext.SetSwitch("Npgsql.EnableLegacyTimestampBehavior", true);
```

Comment:
do we need this?

Reply:
Removed in 3433ebb189a94a9d3bb6218e2ea13e0aeb70a9f3